datadog_checks_base/datadog_checks/base/utils/db/utils.py

```diff
@@ -351,3 +353,21 @@ def _run_job_traced(self):
 
     def run_job(self):
         raise NotImplementedError()
+
+
+@contextlib.contextmanager
+def tracked_query(check, operation, tags=None, track_operation_time=True):
+    """
+    A simple context manager that tracks the time spent in a given query operation
+    """
+    if not track_operation_time:
+        yield
+        return
+    start_time = get_timestamp()
+    stats_kwargs = {}
+    if hasattr(check, 'debug_stats_kwargs'):
+        stats_kwargs = dict(check.debug_stats_kwargs())
+    stats_kwargs['tags'] = stats_kwargs.get('tags', []) + ["operation:{}".format(operation)] + (tags or [])
+    yield
+    elapsed_ms = (get_timestamp() - start_time) * 1000
+    check.histogram("dd.{}.operation.time".format(check.name), elapsed_ms, **stats_kwargs)
```

Comment:
```suggestion
    check.histogram("dd.{}.operation.time".format(check.name), elapsed_ms, raw=True, **stats_kwargs)
```

I'm not familiar with the context or naming conventions for DBM metrics, but if you want to ensure that this name stays this way regardless of the `__NAMESPACE__` attribute of the `check`, you probably want to add `raw=True`.

It would also be useful if we could add more context to the docstring, there's no enough info in here to know why I would be using this context manager, since it appears that these metrics are not meant to be user-facing given this naming convention.